### PR TITLE
LPS-86866 Included two new unit tests

### PR DESCRIPTION
**Pull request by renatodepaiva**

### CODE REVIEW

**Comment by renatodepaiva** [comment]
ci:test:sf

**Comment by renatodepaiva** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/helper/DDMFormEvaluatorHelper.java [hunk 1]
```diff
@@ -303,8 +311,10 @@ protected boolean getBooleanPropertyValue(
 		return _ddmFormFieldsPropertyChanges;
 	}
 
-	protected DDMFormFieldValidation getDDMFormFieldValidation(String field) {
-		DDMFormField ddmFormField = _ddmFormFieldsMap.get(field);
+	protected DDMFormFieldValidation getDDMFormFieldValidation(
+		DDMFormEvaluatorFieldContextKey field) {
```
[the remark this answers is not in the document]
Comment (rafaprax) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/helper/DDMFormEvaluatorHelper.java]: Hey @renatodepaiva 
 Nevermind, I'm going to rename in the review process. Thx

**Comment by rafaprax** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/helper/DDMFormEvaluatorHelper.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (renatodepaiva) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/helper/DDMFormEvaluatorHelper.java]: Done!

**Comment by rafaprax** [comment]
Pull request submitted to https://github.com/rafaprax/liferay-portal/pull/931. [See changes here.](https://github.com/rafaprax/liferay-portal/compare/dca6c8781f5d~1...dca6c8781f5d) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**